app.js

```diff
@@ -3,88 +3,117 @@
+  console.log(storeNameEl);
+  storeNameEl.textContent = this.name;
+  hourlySalesRowEl.appendChild(storeNameEl);
+
+  for(var hours = 0; hours < this.cookiesPerHour.length; hours++){
+    var hourlyCookieSales = document.createElement('td');
+    console.log(hourlyCookieSales);
+    hourlyCookieSales.textContent = this.cookiesPerHour[hours];
+    console.log(hourlyCookieSales);
+    hourlySalesRowEl.appendChild(hourlyCookieSales);
+  }
+  tableToDrawInto.appendChild(hourlySalesRowEl);
+};
+
+var tableEl = document.createElement('table');
+console.log(tableEl);
+var sectionEl = document.getElementById('cookieSalesTable');
+sectionEl.appendChild(tableEl);
+
+var hoursRowEl = document.createElement('tr');
+var hoursHeaderEl = document.createElement('th');
+hoursRowEl.appendChild(hoursHeaderEl);
+for(var tableHours = 0; tableHours < storeHours.length; tableHours++){
+  var hoursDataEl = document.createElement('td');
+  hoursDataEl.textContent = storeHours[tableHours];
+  hoursRowEl.appendChild(hoursDataEl);
+}
+tableEl.appendChild(hoursRowEl);
+
+var stores = [firstAndPike, seatacAirport, seattleCenter, capitolHill, alki];
+
+console.log('==============EVENT LISTENERS===============');
+
+var newUserStores = [];
+var storeFormEl = document.getElementById('new-store-form');
+
+storeFormEl.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event){
+  event.preventDefault();
+  event.stopPropagation();
+
+  var name = event.target.newStoreName.value;
+  var minCustomers = event.target.minCustomers.value;
+  var maxCustomers = event.target.maxCustomers.value;
+  var avgCookies = event.target.avgCookies.value;
+
+  console.log(name);
+  console.log(minCustomers);
+  console.log(maxCustomers);
+  console.log(avgCookies);
+
+  var newStore = new CookieStore(name, minCustomers, maxCustomers, avgCookies);
+  stores.push(newStore);
+  getCookiesPerHour(newStore);
+  tableEl.deleteRow(-1);
+  newStore.makeTheTable(tableEl);
+  makeTotalRow();
+}
+
+for(var storeNumber = 0; storeNumber < stores.length; storeNumber++){
+  getCookiesPerHour(stores[storeNumber]);
+  stores[storeNumber].makeTheTable(tableEl);
+}
+
+function makeTotalRow(){
+  var totalRowEl = document.createElement('tr');
+  tableEl.appendChild(totalRowEl);
+  var totalHeaderEl = document.createElement('th');
+  totalHeaderEl.textContent = 'Totals';
+  totalRowEl.appendChild(totalHeaderEl);
+  var totalCookies = 0;
+  for(var i = 0; i < storeHours.length; i++){
+    for(var ii = 0; ii < stores.length; ii++){
+      totalCookies = totalCookies += stores[ii].cookiesPerHour[i];
+    }
+    var totalDataEl = document.createElement('td');
+    totalDataEl.textContent = totalCookies;
+    totalRowEl.appendChild(totalDataEl);
+    var totalCookies = 0;
   }
 }
 
-//call the functions to make the lists
-putSalesProjections(firstAndPike);
-putSalesProjections(seatacAirport);
-putSalesProjections(seattleCenter);
-putSalesProjections(capitolHill);
-putSalesProjections(alki);
+makeTotalRow();
```

Comment:
If you break out the assorted pieces of functionality distributed throughout the file into functions, then you can group all the function calls down at the bottom--it'll be super readable!